### PR TITLE
App Complete Sample CSharp Fixes

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

**Comment by Mohammed-MSFT** [comment]
Executed Build Pipeline : https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8773082406

**Review by Pawank-MSFT — COMMENTED** [review]
some comments please review and fix.

File: samples/app-complete-sample/csharp/AppCompleteSample/Images/Roaster.png
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/app-complete-sample/csharp/AppCompleteSample/Images/Roaster.png:1]: fixed

File: samples/app-complete-sample/csharp/AppCompleteSample/appsettings.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
   "MicrosoftAppPassword": "{{MicrosoftAppPassword}}",
   "BaseUri": "{{BaseUri}}",
   "MaxComposeExtensionHistoryCount": "10",
-  "FBConnectionName": "{{FBConnectionName}}",
+  "FBConnectionName": "",
```
[the remark this answers is not in the document]
Comment (Mohammed-MSFT) [reply samples/app-complete-sample/csharp/AppCompleteSample/appsettings.json:7]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8890061081

**Pull request merged**